Automattic/newspack-popups
# chore: use newspack-scripts

### All Submissions:






### Changes proposed in this Pull Request:



Update CI images and use `newspack-scripts` as implemented in https://github.com/Automattic/newspack-plugin/pull/1368 and https://github.com/Automattic/newspack-new-plugin-boilerplate/pull/3.

### How to test the changes in this Pull Request:

1. Make sure build and lint commands execute successfully:
   1. `npm start` (this is an alias for `npm ci && npm run watch`)
   2. `npm run build`
   3. `npm run lint:js`
   4. `npm run lint:scss`
2. (**extra step for Campaigns plugin**) Make sure JS tests run and pass: `npm run test`
3. Verify commitizen works by staging a change and running `npm run cm` – a prompt for a structured commit message should appear, just line on master
4. Verify the `commit-msg` hook does lint the commit message:
   1. reinstall the hook – `composer install`
   2. stage a change and try to commit with an unstructured commit message (e.g. `git commit -m "hehe"`) and observe it failing.

### Other information:
